cordapp-example/workflows-kotlin/src/test/kotlin/com/example/test/flow/IOUPaybackFlowTests.kt

```diff
@@ -12,16 +15,21 @@ import net.corda.testing.node.StartedMockNode
 import net.corda.testing.node.TestCordapp
 import org.junit.After
 import org.junit.Before
-import org.junit.Ignore
 import org.junit.Test
+import kotlin.test.assertEquals
 import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
 
 class IOUPaybackFlowTests {
     private lateinit var network: MockNetwork
+    private lateinit var bank: StartedMockNode
     private lateinit var lender: StartedMockNode
     private lateinit var borrower: StartedMockNode
     private lateinit var iouInput: IOUState
 
+    private lateinit var validFlow: PaybackFlow.Initiator
+    private lateinit var validFuture: CordaFuture<SignedTransaction>
```

Comment:
I would create a private function that returns those valid flows instead of having them as fields.